data/mainnet/tokens.js

```diff
@@ -8854,4 +8854,151 @@ module.exports = {
     rowTooltip: 'Out of Range',
     disableAutoSwap: true,
   },
+  aura_auraBAL: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: [
+      VAULT_CATEGORIES_IDS.AURA,
+      VAULT_CATEGORIES_IDS.BALANCER,
+      VAULT_CATEGORIES_IDS.ETH20,
+    ],
+    displayName: 'auraBAL',
+    subLabel: 'Aura Finance',
+    apyIconUrls: ['./icons/balancer.png', './icons/aura.png', './icons/bbausd.png'],
+    apyTokenSymbols: ['BAL', 'AURA', 'bbaUSD'],
+    logoUrl: './icons/aura - auraBAL.png',
+    tokenAddress: addresses.V2.aura_auraBAL.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.V2.aura_auraBAL.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.COINGECKO_ID,
+      params: ['aura-bal'],
+    },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.AURA,
+        params: ['auraBAL', profitSharingCut15Percent],
+      },
+    ],
+    cmcRewardTokenSymbols: ['FARM', 'BAL', 'AURA', 'bbaUSD'],
+  },
+  aura_auraBAL_lp: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: [VAULT_CATEGORIES_IDS.AURA, VAULT_CATEGORIES_IDS.BALANCER],
+    displayName: 'B80BAL-20WETH - auraBAL',
+    subLabel: 'Aura Finance',
+    apyIconUrls: ['./icons/balancer.png', './icons/aura.png'],
+    apyTokenSymbols: ['BAL', 'AURA'],
+    logoUrl: './icons/balancer-auraBALlp.png',
+    tokenAddress: addresses.V2.aura_auraBAL_lp.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.V2.aura_auraBAL_lp.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.BALANCER,
+      params: [
+        addresses.V2.aura_auraBAL_lp.Underlying,
+        addresses.V2.aura_auraBAL_lp.PoolId,
+        CHAINS_ID.ETH_MAINNET,
+      ],
+    },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.AURA,
+        params: ['auraBAL_lp', profitSharingCut15Percent],
+      },
+    ],
+    cmcRewardTokenSymbols: ['FARM', 'BAL', 'AURA'],
+  },
+  aura_bbaUSD: {
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: [
+      VAULT_CATEGORIES_IDS.AURA,
+      VAULT_CATEGORIES_IDS.BALANCER,
+      VAULT_CATEGORIES_IDS.STABLECOINS,
+    ],
+    displayName: 'bb-a-USD',
```

Comment:
Let's split this up into the underlying tokens, `aDAI-aUSDC-aUSDT`

Reply:
Fixed